flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/EmbeddedRocksDBStateBackendTest.java

```diff
@@ -610,6 +612,10 @@ public void testSharedIncrementalStateDeRegistration() throws Exception {
         }
     }
 
+    @Override
+    @Test
+    public void testMaterializedRestore() {}
+
```

Comment:
This override prevents the test from running at all, even for changelog backend tests.

Reply:
The delegated state backend does not have materialization; that's why the test is ignored.

Reply:
Yes, but with this override even the changelog backend isn't tested: its tests inherit this empty method.

Reply:
LOL, that's true! Thanks for point this out!